easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenJPEG'
+version = '2.3.0'
+
+homepage = 'http://www.openjpeg.org/'
+description = """OpenJPEG is an open-source JPEG 2000 codec written in C language. It has been developed in order
+to promote the use of JPEG 2000, a still-image compression standard from the Joint Photographic Experts Group
+(JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as a JPEG 2000 Reference Software."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/uclouvain/openjpeg/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['3dc787c1bb6023ba846c2a0d9b1f6e179f1cd255172bde9eb75b01f1e6c7d71a']
+
+builddependencies = [
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.30', '', True),
```

Comment:
@crubb No need to use dummy binutils here, please use ('binutils', '2.30') instead